internal/pkg/agent/application/upgrade/step_mark.go

```diff
@@ -103,16 +155,31 @@ func LoadMarker() (*UpdateMarker, error) {
 		return nil, err
 	}
 
-	marker := &UpdateMarker{}
+	marker := &updateMarkerSerializer{}
```

Comment:
will this work if marker already contains new form of action ID?

Reply:
the action_id breakage is new and specific to 8.3 branch, it was not releases as a part of the 8.2.x, I tested the upgrade between latest 8.2.4 and 8.3